sympy/core/sympify.py

```diff
@@ -260,20 +299,7 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     # Support for basic numpy datatypes
     if type(a).__module__ == 'numpy':
         import numpy as np
```

Comment:
Can this line be removed too?

Reply:
I'm not really sure. I think the first time I factored the function out it yelled at me about not knowing what ```numpy``` was. Maybe there's a better way to do this?